dsc/src/resource_command.rs

```diff
@@ -177,34 +216,25 @@ pub fn export(dsc: &mut DscManager, resource: &str, format: &Option<OutputFormat
     write_output(&json, format);
 }
 
-pub fn get_resource(dsc: &mut DscManager, resource: &str) -> DscResource {
+#[must_use]
+pub fn get_resource<'a>(dsc: &'a DscManager, resource: &str) -> Option<&'a DscResource> {
```

Comment:
not clear to me why lifetime annotation is needed here

Reply:
"missing lifetime specifier
this function's return type contains a borrowed value, but the signature does not say whether it is borrowed from `dsc` or `resource`"